core/declarations.gms

```diff
@@ -552,8 +552,8 @@ p_emi_budget1_gdx                                     "budget for global energy-
 s_reference2030co2eq                                  "reference level of 2030 GHG emissions for AWP2 myopic scenarios in GtCO2eq p.a., all Kyoto gases"
 s_referencebudgetco2                                  "reference level of 2000-2100 cumulated emissions for AWP2 myopic scenarios in GtCO2, including all CO2"
 s_actual2030co2eq                                     "actual level of 2030 GHG emissions for AWP2 myopic scenarios in GtCO2eq p.a., all Kyoto gases for last iteration"
-s_actualbudgetco2                                     "actual level of 2000-2100 cumulated emissions for AWP2 myopic scenarios in GtCO2, including all CO2 for last iteration"
-s_actualbudgetco2_last                                "actual level of 2000-2100 cumulated emissions for previous iteration" /0/
+s_actualbudgetco2                                     "actual level of 2020-2100 cumulated emissions for AWP2 myopic scenarios in GtCO2, including all CO2 for last iteration"
```

Comment:
haha, that's really old! nobody knows anymore what AWP2 myopic scenarios are. :) we can just delete the "for AWP2 myopic scenarios".